R/car.R

```diff
@@ -74,7 +74,7 @@ scale_gmrf_precision <- function(Q,
                                  A = matrix(1, ncol = ncol(Q)),
                                  eps = sqrt(.Machine$double.eps)) {
 
-  nb <- spdep::mat2listw(abs(Q), style = "M")$neighbours
+  nb <- spdep::mat2listw(abs(Q), style = "B")$neighbours
```

Comment:
Note I've updated this using Paddy's fix from https://github.com/mrc-ide/naomi/pull/391 which should stop the huge amount of warnings. We can close that PR after this has been merged

Reply:
Actually tried that and it broke everything, just suppressing this warning on "M" for now as I can't find an equivalent to this which won't trigger a warning